source/modules/MEI.visual.xml

```diff
@@ -679,6 +679,18 @@
           <rng:ref name="data.BOOLEAN"/>
         </datatype>
       </attDef>
+      <attDef ident="opening.vertical" usage="opt">
+        <desc xml:lang="en">Indicates if the opening points must be aligned vertically, including when the hairpin is angled with @optimize.angle or through @endvo/@startvo adjustments.</desc>
```

Comment:
I think what is meant is that the points should be connected with a line that is always 90° perpendicular to the horizontal plane, regardless of any angle or start/end adjustments. As it is, just saying that the points should be "aligned vertically" doesn't say how they are to be aligned.

Reply:
Except that the points are not connected. So I would suggest
```
Indicates if the opening points must be aligned with a line that is always 90° perpendicular to the horizontal plane, regardless of any angle or start/end adjustments, including when the hairpin is angled with @optimize.angle or through @endvo/@startvo adjustments.
```
Better?

Reply:
Sorry, in an earlier draft I had "an imaginary line" but that got missed in the editing.

Reply:
```
Indicates that the opening points must be aligned with an imaginary line that is always 90° perpendicular to the horizontal plane, regardless of any angle or start/end adjustments, including when the hairpin is angled with @optimize.angle or through @endvo/@startvo adjustments.
```